app/build.gradle

```diff
@@ -77,7 +77,7 @@ dependencies {
     ksp 'io.github.raamcosta.compose-destinations:ksp:1.5.3-beta'
 
     // euphony
-    implementation 'euphony.lib:euphony:0.7.1.6'
+    implementation 'euphony.lib:euphony:0.8'
```

Comment:
here has some version update commit in here.
Actually, this will make some build error :( and it is not related to change package name.

Reply:
Right :) So, only add commits related to ```change package name``` 😄